core/src/instance.rs

```diff
@@ -161,18 +162,19 @@ impl Instance {
         rx_action: Receiver<ActionWrapper>,
         rx_observer: Receiver<Observer>,
     ) {
-        let sync_self = self.clone();
+        let mut sync_self = self.clone();
```

Comment:
Necessary because of the stateful nature of the `ConsistencyModel` (it needs to keep track of Commits so that it can emit that data when the entry is later Published)